public/index.html

```diff
@@ -20,11 +20,8 @@
 <html>
 
 <head>
-    <!-- // TODO: need to copy this automatically as poart of the build
```

Comment:
probably we could leave a comment explaining why or how... Probably 1 sentence with a link is enough

Reply:
I already put one new sentence. Is it not descriptive enough?

Reply:
I would add a link so the next developer can understand why this is necessary.. for example  https://firebase.google.com/docs/perf-mon/get-started-web  (is it the right one)

Reply:
updated